protocol-designer/src/components/modals/AnnouncementModal/announcements.tsx

```diff
@@ -190,4 +195,28 @@ export const announcements: Announcement[] = [
       </>
     ),
   },
+  {
+    announcementKey: 'flexSupport7.0',
+    image: (
+      <Flex justifyContent={JUSTIFY_CENTER}>
+        <img
+          height="240"
+          width="240"
```

Comment:
just make things clear (I believe everyone can understand the unit  lol)
```suggestion
          height="240px"
          width="240px"
```